documentation/how-to/attributes-form.en.md

```diff
@@ -43,6 +43,12 @@ QField form.
 
 Groups and Tabs from Drag and Drop designer forms are supported.
 
+In the "Attributes Form," you can structure your forms using various containers, such as tabs and groups, and enhance interactivity by incorporating expressions with conditionals. By enabling the drag-and-drop designer, you gain the flexibility to display attribute fields, as well as other widgets that are not directly tied to specific fields (e.g., HTML/QML widgets) and "Relations" of layers.
```

Comment:
You should also add a link to QGIS' own documentation here: https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/vector_properties.html#vector-attributes-menu

```suggestion
In the "Attributes Form," you can structure your forms using various containers, such as tabs and groups, and enhance interactivity by incorporating expression-based conditional visibility. By enabling the drag-and-drop designer, you gain the flexibility to re-order attribute fields, as well as gaining access to widgets that are not directly tied to specific fields (e.g., HTML/QML widgets).
```